examples/README.md

```diff
@@ -0,0 +1,182 @@
+# Iceberg Java API Examples (with Spark)
+
+## About
+Welcome! :smile:
+
+If you've stumbled across this module, hopefully you're looking for some guidance on how to get started with the [Apache Iceberg](https://iceberg.apache.org/) table format. This set of classes collects code examples of how to use the Iceberg Java API with Spark, along with some extra detail here in the README.
+
+The examples are structured as JUnit tests that you can download and run locally if you want to mess around with Iceberg yourself. 
+
+## Running Iceberg yourself
+If you'd like to try out Iceberg in your own project, you can use the `spark-iceberg-runtime` dependency:
```

Comment:
Looks like this assumes that the use of Iceberg is going to be in a maven project, right?

We already have some docs on how to include Iceberg to make it available with Spark itself (add it to jars) or to submit it with a job (use `--packages`) here: https://iceberg.apache.org/spark/#spark-24

Should we refactor that section and this one in a new docs page with a section for each case? I'm thinking compilation in a Maven Spark project and a gradle Spark project, then runtime using `--packages` or a provided Iceberg within Spark by adding the runtime to Spark's `jars` folder.

Reply:
For this PR, maybe just note to clarify that this recommendation is for creating a maven project. Might want to add gradle here, too.

Reply:
Yep, this assumes the use of Maven. I think adding a new page for how to include Iceberg in different types of projects sounds like a good idea! I'll get a page ready to add to this PR :) I'm thinking the page should go under the User docs tab?